pkg/controller/tas/node_failure_controller_test.go

```diff
@@ -84,6 +84,10 @@ func TestNodeFailureReconciler(t *testing.T) {
 		Admitted(true).
 		Obj()
 
+	workloadWithAnnotation := baseWorkload.DeepCopy()
+	workloadWithAnnotation.Annotations = make(map[string]string)
+	workloadWithAnnotation.Annotations[kueuealpha.NodeToReplaceAnnotation] = nodeName
```

Comment:
```suggestion
	workloadWithAnnotation.Annotations := map[string]string{
		kueuealpha.NodeToReplaceAnnotation: nodeName,
	}
```

nit: I think we can directly initialize this map.